### PR TITLE
Marios development

### PR DESCRIPTION
**Pull request by mariom3**
Finish up bare minimums of portfolio.

### CODE REVIEW

File: portfolio/src/main/webapp/css/aboutme.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+html, body {
+    height: 100%;
+    width: 100%;
+}
+
+body{
```
Comment (sbahaud) [inline portfolio/src/main/webapp/css/aboutme.css]: nit: Please add a space before {

File: portfolio/src/main/webapp/css/aboutme.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    display: flex;
+    flex-direction: column;
+    font-family: Garamond;
+    height: 100%;
+    width: 100%;
+}
+
+#top-ref {
+    width:100%;
+    height: auto;
```
Comment (sbahaud) [inline portfolio/src/main/webapp/css/aboutme.css]: Please order the properties alphabetically

File: portfolio/src/main/webapp/css/aboutme.css (hunk 1 above)
Comment (mariom3) [reply portfolio/src/main/webapp/css/aboutme.css]: Done.

File: portfolio/src/main/webapp/css/aboutme.css (hunk 2 above)
Comment (mariom3) [reply portfolio/src/main/webapp/css/aboutme.css]: Done: Removed since it was not in use.

**Review by sbahaud — APPROVED** [review]
LGTM

**Pull request merged**